templates/compose/linkding-plus.yaml

```
@@ -0,0 +1,20 @@
+# documentation: https://linkding.link/
+# slogan: A self-hosted bookmark manager designed to be minimal, fast, and easy to set up. (Includes feature for archiving websites as HTML snapshots)
+# category: RSS
+# tags: rss, feed
+# logo: svgs/linkding.svg
+# port: 9090
+
+services:
+  linkding:
+    image: sissbruecker/linkding:latest-plus
+    volumes:
+      - './data:/etc/linkding/data:rw'
+    environment:
+      - SERVICE_URL_LINKDING_9090
+      - 'LD_SUPERUSER_NAME=${SUPERUSER_NAME}'
+      - 'LD_SUPERUSER_PASSWORD=${SUPERUSER_PASSWORD}'
```

These variables seems like they are required for the service to work so we can mark them as Required. Required variables must be set before deployment and will be highlighted in Coolify's UI with a red border if empty.

```suggestion
      - 'LD_SUPERUSER_NAME=${SUPERUSER_NAME:?}'
      - 'LD_SUPERUSER_PASSWORD=${SUPERUSER_PASSWORD:?}'
```

---

I was going back and forth on this: The super user name and password are not technically needed - if you don't provide them when starting the service for the first time, you can't sign in. But once you sign in, the environment variables are not taken into consideration anymore and you could delete them from the docker compose file and from the environment variables, and the service would still work.

Then again, this shouldn't be a problem - unless you don't change the password in the application UI and someone gains access to your coolify UI.

That being said, I'll follow your recommendation if could confirm it 🙏🏻